src/org/thoughtcrime/securesms/crypto/KeyStoreHelper.java

```diff
@@ -170,7 +167,7 @@ public static class SealedData {
     }
 
     @SuppressWarnings("unused")
-    public SealedData() {}
+    public SealedData() { /* needed by JsonUtils.fromJson() */ }
```

Comment:
if it is needed and used - why then suppress "unused" warning?

Reply:
because it is used indirectly, to JsonUtils.fromJson() SealedData.class is passed, then some json marshal/unmarshal lib uses the class with some reflection to parse json and create instances of the given class, so the IDE can't really know it is actually used via reflection

Reply:
thanks for explanation. then removing "unused" things seems dangerous, if IDE does not really know (just saying in general, this PR seems to be fine as mostly TAG things are removed :)

Reply:
I only removed the "unused" annotation where it was not needed (because there was obvious usages in other places and the IDE didn't complain) removing it here for example, would make the IDE then create a warning, so what I did was remove the false positives so it is more clear where there are real "unused" annotations and that they are important and not just leftovers